core/modules/wbase/Task.h

```diff
@@ -58,7 +58,7 @@ namespace wbase {
 /// Task encapsulates nearly zero logic, aside from:
 /// * constructors
 /// * poison()
-struct Task : public boost::noncopyable {
+struct Task {
```

Comment:
It still not copyable. You can now define copy constructor and equal operator = delete, which makes it non-copyable by prevent default constructors from being used. Lines 79 and 80.